helm/cluster/values.yaml

```diff
@@ -31,10 +31,16 @@ global:
         docker.io:
           - endpoint: registry-1.docker.io
           - endpoint: giantswarm.azurecr.io
```

Comment:
@piontec Should we get rid of `giantswarm.azurecr.io` on CAPx releases?

Reply:
I left it here on purpose, because removing it would be a breaking change. I'm not sure if something somewhere relies on this. In any case, with a breaking change we would have to release this in a new major release, and now with v27 being already out, that would mean releasing this in v28 the soonest. And IIUC we want this in v25, in a new minor, so newly migrated AWS clusters get to use this. A new v25 minor cannot have a breaking change.

IMO we can remove this later, no harm in keeping the old defaults, while adding the new ones.